### PR TITLE
Simplify the preparation for JCL testing

### PR DESCRIPTION
**Pull request by std4lqi**
**What It Does**
Remove JCL preparation from `default_properties.yaml`, but have test case to check and upload test JCL when needed. This will make it easy for tester do prepare the environment (datasets and uss directories) on z/OS to run system tests.

**How to Test**
Run test case under `__tests__/__system__/cli`. 

**Review Checklist**
I certify that I have:
- [x] tested my changes
- [x] added/updated automated tests
- [x] updated the changelog
- [x] followed the [contribution guidelines](https://github.com/zowe/zowe-cli/blob/master/CONTRIBUTING.md)

**Additional Comments**
N/A

### CODE REVIEW

**Review by tiantn — APPROVED** [review]
I run unit test and system test. All passed. Thank you!

**Review by JillieBeanSim — APPROVED** [review]
LGTM! thanks for the fix @std4lqi

**Comment by std4lqi** [comment]
@JillieBeanSim , @zFernand0 are we OK to merge it with the code duplication check failure on testing code?

**Comment by zFernand0** [comment]
> @JillieBeanSim , @zFernand0 are we OK to merge it with the code duplication check failure on testing code?

yeah, a lot of times we end up with some duplication due to how tests are structured.
I wouldn't worry too much about code duplication.
image

File: __tests__/__resources__/properties/default_properties.yaml
Comment (zFernand0) [inline __tests__/__resources__/properties/default_properties.yaml:1]: thanks for adding comments to the properties in this file 😋

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,12 @@
 
 All notable changes to the z/OS FTP Plug-in for Zowe CLI will be documented in this file.
 
+
+## Recent Changes
+
+- Simplify the preparation for JCL system tests
```
Comment (zFernand0) [inline CHANGELOG.md:8]: not sure this requires a new patch to be published.
However we can combine the contents of this PR (after merged) with:
- #152

**Review by zFernand0 — APPROVED** [review]
LGTM! 😋

**Pull request merged**